neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Neo.SmartContract.Native.Tokens
+{
+    public sealed class GasToken : Nep5Token<Nep5AccountState>
+    {
+        public override string ServiceName => "Neo.Native.Tokens.GAS";
+        public override string Name => "GAS";
+        public override string Symbol => "gas";
+        public override int Decimals => 8;
+
+        private GasToken()
```

Comment:
Isn't it strange that the Activator can create an instance of GasToken even if it's private? Just wondering...

Reply:
Yes, doesn't matter